.github/workflows/jit.yml

```
@@ -115,7 +115,7 @@ jobs:
           find /usr/local/bin -lname '*/Library/Frameworks/Python.framework/*' -delete
           brew install llvm@${{ matrix.llvm }}
           export SDKROOT="$(xcrun --show-sdk-path)"
-          ./configure --enable-experimental-jit ${{ matrix.debug && '--with-pydebug' || '' }}
+          ./configure --enable-experimental-jit --enable-universal-sdk --with-universal-archs=universal2 ${{ matrix.debug && '--with-pydebug' || '' }}
```

Typo? I *think* this is spelled `--enable-universalsdk`:

```suggestion
          ./configure --enable-experimental-jit --enable-universalsdk --with-universal-archs=universal2 ${{ matrix.debug && '--with-pydebug' || '' }}
```

---

Yes, lol. I was fussing with this ate last night and could not figure out why this wasn't working...